test/client/fixtures/runner/runner-test.js

```diff
@@ -1,35 +1,63 @@
-var hammerhead  = window.getTestCafeModule('hammerhead');
+var hammerhead = window.getTestCafeModule('hammerhead');
 var hhsettings = hammerhead.get('./settings').get();
 
 var testCafeCore = window.getTestCafeModule('testCafeCore');
 var COMMAND      = testCafeCore.COMMAND;
 var SETTINGS     = testCafeCore.get('./settings').get();
 var transport    = testCafeCore.get('./transport');
+var ERROR_TYPE   = testCafeCore.ERROR_TYPE;
 
 var testCafeRunner = window.getTestCafeModule('testCafeRunner');
 var actionBarrier  = testCafeRunner.get('./action-barrier/action-barrier');
 var Runner         = testCafeRunner.get('./runner');
 
 
-transport.batchUpdate                = function (callback) {
+var runner                        = null;
```

Comment:
as I see the `runner` var is not used. You use `testRunner` below. I suggest to rename `testRunner` to `runner`